### DISCUSSION TITLE
Feature request to set and optional Username:Password for RTSP Stream?

### DISCUSSION THREAD

**Discussion by Richard-G4WKW** [iamscottxu/obs-rtspserver#9] [Ideas]
Thank you for developing your plugin, I am already delighted in what it has enabled me to do, in particular, streaming to the Internet a live feed from OBS Studio. 

I am already using https://rtsp.me to stream from an IP camera (my BirdCam) on my LAN using RTSP with the site providing a link that can display the feed in a web browser.  To do this, I used IPv4 Port Mapping on my router to set up a map for Port 554 on the camera LAN IP to the same port on the WAN IP.   The camera (which is a TP-LINK Tapo type needed me to set up a camera account username and password and the stream from the camera was then in the format:

rtsp://username:password@IPAddress:554/stream1

I could use Network Streaming in VLC to view the camera using either the LAN or WAN IP address.

Anyway, I set up RTSP streaming from OBS using your plugin, using port 8554, and set up IPv4 Port Mapping (just TCP) again on my router and set up a test cam feed in https://rtsp.me, using the format:

rtsp://IPAddress:8554/live

This all worked fine, but I would have preferred if I could have used a username and password on the stream so that I could have used this format instead:

rtsp://username:password@IPAddress:8554/live

It would be nice if you could implement an optional Username and Password to be set for your RTSP Server to add some security.

**Comment by medybala** [comment]
Hello, thank you for developing this plugin. Is it possible to make a field with the inscription "live" in the address rtsp
 editable?

**Comment by iamscottxu** [comment]
@Richard-G4WKW Now, you can use username and password authentication in version 2.0.0.

**Reply by medybala** [reply]
version 2.0.0 is ok with username and password, but i can't change the "live" word.
